frontend/static/messages/pl.json

```
@@ -3287,10 +3294,18 @@
         "actionPush": "Powiadomienie push",
         "optionsSection": "Opcje",
         "cooldownMinutes": "Czas odnowienia (minuty)",
+        "cooldownSeconds": "Czas odnowienia (sekundy)",
```

Biome (2.4.4)

[error] The key cooldownSeconds was already declared.

(lint/suspicious/noDuplicateObjectKeys)